### PR TITLE
LIKUNTA-115 | Add accessibility statement and about us pages

### PR DESCRIPTION
**Pull request by tuovinensanttu**
This PR adds accessibility statement and about us pages. I'm still wondering if I should have created one component that eats `PAGE_QUERY` as the data is almost identical between the pages 🤔.

### CODE REVIEW

File: src/pages/accessibility/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    variables: {
+      languageCode: language,
+    },
+  });
+
+  return {
+    props: {
+      initialApolloState: cmsClient.cache.extract(),
+      ...(await serverSideTranslationsWithCommon(context.locale, [
+        "home_page",
```
Comment (Jokinen) [inline src/pages/accessibility/index.tsx:73]: Unused translations.

File: src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -72,8 +72,11 @@ function Footer({ navigationItems }: Props) {
         copyrightHolder={t("copyright_holder")}
         copyrightText={t("copyright_text")}
       >
-        <HDSFooter.Item href="#" label={t("about_us")} />
-        <HDSFooter.Item href="#" label={t("accessibility_statement")} />
+        <HDSFooter.Item href="/about" label={t("about_us")} />
+        <HDSFooter.Item
+          href="/accessibility"
+          label={t("accessibility_statement")}
+        />
```
Comment (Jokinen) [inline src/components/footer/Footer.tsx:79]: Using `as={LInk}` would translate the paths they point to (which you configured in the `i18nRoutes.config.js` file).

**Review by Jokinen — APPROVED** [review]
The components could be combined. The slug for them could be set as dynamic and provided as a variable

```
  query PageQuery($languageCode: LanguageCodeEnum!, $uri: !String) {
    page(id: $uri, idType: URI) {
```

It would introduce some extra considerations: how to ensure that there are no URL collisions, and if there are, how do we resolve them. For instance, perhaps we should use a `pages` directory for these kind of pages to give them their own namespace.

But it would allow us to define this integration once and would guide us to maintain the integration in a way that is in line with the thinking of the headless WP API (as far as I am aware).

Personally I would consider combining the pages if time allows--better to have less code to maintain. But I am also fine with merging this PR as it is, and combining the pages later when we know more.

**Comment by Jokinen** [comment]
One thing I am thinking is whether we should have moved this particular menu into the CMS at this point: https://liikunta.content.api.hel.fi/wp-admin/nav-menus.php?action=locations

Perhaps we could have used a tertiary menu for this? Not entirely sure about the purpose for it. I assume that primary is the main menu and secondary is the menu in the footer.

File: src/pages/about/index.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  );
+}
+
+export async function getStaticProps(context: GetStaticPropsContext) {
+  const cmsClient = initializeCmsApollo();
+  const language = getQlLanguage(context.locale ?? context.defaultLocale);
+
+  await cmsClient.pageQuery({
+    nextContext: context,
+    query: LANDING_PAGE_QUERY,
```
Comment (Jokinen) [inline src/pages/about/index.tsx:63]: Incorrect query--breaks caching.

File: src/pages/accessibility/index.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  );
+}
+
+export async function getStaticProps(context: GetStaticPropsContext) {
+  const cmsClient = initializeCmsApollo();
+  const language = getQlLanguage(context.locale ?? context.defaultLocale);
+
+  await cmsClient.pageQuery({
+    nextContext: context,
+    query: LANDING_PAGE_QUERY,
```
Comment (Jokinen) [inline src/pages/accessibility/index.tsx:63]: Incorrect query--breaks caching.

**Review by Jokinen — CHANGES_REQUESTED** [review]

File: src/pages/about/index.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    variables: {
+      languageCode: language,
+    },
+  });
+
+  return {
+    props: {
+      initialApolloState: cmsClient.cache.extract(),
+      ...(await serverSideTranslationsWithCommon(context.locale, [
+        "home_page",
```
[the remark this answers is not in the document]
Comment (tuovinensanttu) [reply src/pages/about/index.tsx:73]: Hmm, I'm not sure if I did something wrong. But without this the translations in the main navigation are missing / using translation key.

File: src/pages/about/index.tsx [hunk 6]
```diff
@@ -60,7 +59,7 @@ export async function getStaticProps(context: GetStaticPropsContext) {
 
   await cmsClient.pageQuery({
     nextContext: context,
-    query: LANDING_PAGE_QUERY,
+    query: ACCESSIBILITY_PAGE_QUERY,
```
Comment (Jokinen) [inline src/pages/about/index.tsx:62]: Is this query named in an odd way? Considering that this is the about page?

**Review by Jokinen — APPROVED** [review]

File: src/pages/about/index.tsx (hunk 6 above)
Comment (tuovinensanttu) [reply src/pages/about/index.tsx]: Renamed now. Seems that I can't win with these copy pasted things on this PR 😄

**Pull request merged**